pennylane/devices/qubit/sampling.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2018-2023 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Functions to sample a state."""
+
+import pennylane as qml
+from pennylane import numpy as np
+
+
+def states_to_binary(samples, num_wires: int) -> np.ndarray:
```

Comment:
Do we need to extract this into a helper function?

Reply:
no, I had just done it because I suspected it might be needed elsewhere. But you're right, until it is, why separate? I'll put it back